insights/parsers/tests/test_krb5.py

```diff
@@ -0,0 +1,92 @@
+from insights.parsers import krb5
+from insights.tests import context_wrap
+
+
+KRB5CONFIG = """
+# Configuration snippets may be placed in this directory as well
+includedir /etc/krb5.conf.d/
+
+[logging]
+ default = FILE:/var/log/krb5libs.log
+ kdc = FILE:/var/log/krb5kdc.log
+ admin_server = FILE:/var/log/kadmind.log
+
+[realms]
+ dns_lookup_realm = false
+ auth_to_local = RULE:[1:$1@$0](.*@.*EXAMPLE.ORG)s/@.*//
```

Comment:
Once again, do not make up examples.  auth_to_local does not exist outside a realm.  See `man krb5.conf` for details on how the [realms] section is parsed.  Using bad examples means that we miss implementation details that should be checked.